tools/CI/Dockerfile

```diff
@@ -0,0 +1,135 @@
+FROM ubuntu:22.04
+
+# So tzdata package doesn't ask for user interaction
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install any needed packages to run cmake with full CHIME build options
+RUN apt-get -y update && \
+    apt-get install -y software-properties-common=0.99.* && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y python3.11=3.11.* \
```

Comment:
As we discussed, I think the versions here can be somewhat relaxed for the 22.04 build (although possibly Python will need to stay pinned due to library issues we need to resolve first).   For 18.04 we should keep them locked, since we are keeping support for 18.04 really just to support a few systems which are effectively version locked.